src/pages/MsigUI.vue

```diff
@@ -88,22 +154,95 @@ export default defineComponent({
 			...toRefs(state),
 			key,
 			onTabChange,
+			mparamsPartial,
+			mparamsSigned,
 		};
 	},
 	methods: {
-		sign() {
+		async sign() {
+			let TxnBase64: string;
+			TxnBase64 = "";
+			let signedTxn: JsonPayload;
+			TxnBase64 = this.unsignedJson;
 			switch (this.walletStore.walletKind) {
 				case WalletType.MY_ALGO: {
 					let signMyAlgo = this.walletStore.webMode as MyAlgoWalletSession;
+
+					let Trxs = algosdk.decodeUnsignedTransaction(
+						Buffer.from(TxnBase64, "base64")
+					);
+					let tmpSign = await signMyAlgo.signTransaction(Trxs);
```

Comment:
Why we using tmpSign here? Is this necessary?

Reply:
This one is for testing the feature that we use my algo wallet to sign a transaction. I am waiting for the team myalgo to solve my issuses to continue on this one!